### DISCUSSION TITLE
How do flares work?

### DISCUSSION THREAD

**Discussion by carpediembr** [TEHE-Studios/ExpandedHelicopterEvents#131] [Q&A] [answered]
How do they work?

**Comment by Chuckleberry-Finn** [comment, marked as answer]
The new heatmap system tracks activity within cells, like moving, shooting, killing lots of zombies, firing a flare is much stronger than the other activities.

Some events are also given some logic to switch targets if the "heat" difference is significant enough.

The end result should be that firing flares before or during an event may attract the event to you.

It does not simply spawn an event.
